Client/src/components/login.tsx

```diff
@@ -0,0 +1,17 @@
+import '../css/login.css';
+import { Auth0Provider } from "@auth0/auth0-react";
+
+function Login(props:any) {
+  
+  return(
+    <Auth0Provider
```

Comment:
this file is only necessary IF someone access to the website with the /login, in my opinion. This can be deleted in the future,

Reply:
Done